package.json

```diff
@@ -69,7 +69,7 @@
     "prettier": "^2",
     "semantic-release": "^17",
     "ts-jest": "^26",
-    "typescript": "^4"
```

Comment:
Is there a reason not to specify an exact version number here?

Reply:
🤖 ok we'll do that

Reply:
I think there’s a clearer separation of human 🧑 / robot 🤖  between `package.json` / `yarn.lock`